notebooks/mping_initial_xai.ipynb

```diff
@@ -0,0 +1,5548 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    utils.plot_confusion_matrix(y_test_labels, y_pred_labels, classes=class_names,
</pre><p>There is no utils.py file in the pull request. Please add and commit it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2es/ptype-physical/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>